packages/ngworker/lumberjack/src/lib/log-drivers/lumberjack-log-driver-logger.ts

```diff
@@ -17,12 +17,14 @@ import { LumberjackLogDriverLog } from './lumberjack-log-driver.log';
  */
 @Injectable()
 export class LumberjackLogDriverLogger<TPayload extends LumberjackLogPayload | void = void> {
-  private readonly [LumberjackLevel.Critical] = criticalLogDriverLoggingStrategy;
-  private readonly [LumberjackLevel.Debug] = debugLogDriverLoggingStrategy;
-  private readonly [LumberjackLevel.Error] = errorLogDriverLoggingStrategy;
-  private readonly [LumberjackLevel.Info] = infoLogDriverLoggingStrategy;
-  private readonly [LumberjackLevel.Trace] = traceLogDriverLoggingStrategy;
-  private readonly [LumberjackLevel.Warning] = warningLogDriverLoggingStrategy;
+  #driverLogStrategyMap = {
```

Comment:
issue: Let's keep `readonly`
```suggestion
  readonly #driverLogStrategyMap = {
```